### PR TITLE
tremor, add new recipe for vorbisidec package

### PR DESCRIPTION
**Pull request by Begasus**
I know it's an old one, but wanted to check this out for another issue that was calling for this library ...

### CODE REVIEW

File: media-libs/tremor/tremor-1.0.0~git.recipe [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+vorbisfile library included with the open source Vorbis reference \
+libraries distributed for free by Xiph.org."
+HOMEPAGE="https://wiki.xiph.org/Tremor/"
+COPYRIGHT="1994-2018 Xiph.Org Foundation"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+srcGitRev="4b14e2579e91e22cd5dcc63553496ef6e7c5c222"
+SOURCE_URI="https://github.com/jhot/libtremor/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="f122b4d9f00412929b407c00f19d1b713604d57fb9ba72a98f4a68e77639abd1"
+SOURCE_FILENAME="tremor-$srcGitRev.tar.gz"
```
Comment (fbrosson) [inline media-libs/tremor/tremor-1.0.0~git.recipe]: Suggestion:
```diff
-SOURCE_FILENAME="tremor-$srcGitRev.tar.gz"
+SOURCE_FILENAME="tremor-$portVersion-$srcGitRev.tar.gz"
```

Comment (Begasus) [reply media-libs/tremor/tremor-1.0.0~git.recipe]: Wasn't sure about that one :+1:

File: media-libs/tremor/tremor-1.0.0~git.recipe [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm "$libDir"/libvorbisidec*.la
+
+	mkdir -p "$includeDir"/tremor
```
Comment (korli) [inline media-libs/tremor/tremor-1.0.0~git.recipe]: You could create the $includeDir and move the tremor directory from $prefix/include.

Comment (Begasus) [reply media-libs/tremor/tremor-1.0.0~git.recipe]: with the new patchset not needed anymore :)

**Comment by fbrosson** [comment]
I've checked that it builds fine on all archictures.

BTW, I've pushed a tiny commit with a couple of cosmetic neutral edits. I guess you are OK with it, but if you wish I can remove my commit.

So, shall we merge?

**Comment by Begasus** [comment]
Changes are correct, I'm just not sure about setting the @includedir@/tremor in the *.pc.in file, the latest REminiscence source looks for the header in $includeDir/tremor, but it calls also for "#include tremor/*.h", so I'm not sure setting @includedir@/tremor would be the way to go?

**Comment by fbrosson** [comment]
IDK, I'd have to look closer...
Let's keep this PR open for now, until we find out what is the best answer to your question about the contents of the _.pc_ file.

**Comment by fbrosson** [comment]
@Begasus: I haven't had the time to look closer...
So I don't know if this PR can be merged.
I'll let you decide.

**Pull request merged**